jupyter_book/page/page.py

```diff
@@ -246,7 +246,7 @@ def page_html(ntbk, path_media_output=None, name=None, preprocessors=None,
     html, resources = exp.from_notebook_node(ntbk, resources=output_resources)
 
     html = f"""
-    <main class="jupyter-page">
+    <main class="jupyter-page c-textbook__content">
```

Comment:
this is going to create some formatting weirdness in the actual book (textbook__content is a book class):

https://deploy-preview-426--jupyter-book.netlify.com/intro

better that we:

1. Pick the CSS styles that should be shared between a "book" TOC and a single page TOC, move them into a selector that is agnostic to a book-specific class
2. Define a class that's specific to the single page (maybe just `class="jupyter-page single-page"`?) and add some CSS rules for the one-page case